packages/docx/src/ColorSwatches/ColorSwatch/ColorSwatch.css

```diff
@@ -1,23 +1,27 @@
 .swatch {
   display: inline-block;
-  width: calc(100% / 4 - 0.5rem);
-  margin: 0.25rem;
+  width: 25%;
+  margin: var(--space-small) 0 var(--space-smaller);
+}
+
+.color[style*="background-color: rgb(255, 255, 255);"] {
```

Comment:
What in god's name is he doing, you ask? I removed the borders but that's a problem for "white" since the page background is white. There might be a way to do this in the .tsx but this seemed like the "principle of least power" approach.

Reply:
This makes sense to me for now. Might be worth a comment in the code to explain what is going on here.